app/index.html

```diff
@@ -17,13 +17,31 @@
   <header class="hero">
     {% include 'templates/topnav.html' %}
     <div class="wrapper">
-      <div class="hero-title">Polymer Project</div>
+      <div class="hero-title"><span id="polymer-logo"></span>Polymer Project</div>
+      <!-- <div class="hero-title">Polymer Project</div> -->
       <p class="hero-caption">We work on libraries, tools, and standards to help developers build a better web</p>
       <a class="hero-link link-with-arrow" href="/blog/">Blog</a>
     </div>
   </header>
 
   <style>
+    #polymer-logo {
```

Comment:
Prefer class over id, even if only a single instance on page.

Why `<span>` with background-image instead of `<img>` tag? Prefer `<img>` unless there's a good reason, like CSS limitations.

Move this entire `<style>` tag above `<header>` so that all the elements it styles are below the `<style>` tag.